addons/web/static/tests/views/fields/statusbar_time_field_tests.js

```diff
@@ -0,0 +1,91 @@
+/** @odoo-module **/
+
+import { getFixture } from "@web/../tests/helpers/utils";
```

Comment:
Why are the statusbar tests in web and not in mail ? Unless I misunderstood something ?

Reply:
Just some leftovers from when the mixin mas in web, modified it